atomdb/test/test_gaussian.py

```diff
@@ -32,6 +32,91 @@
 TEST_DATAPATH = "atomdb/test/data/"
 
 
+@pytest.mark.dev
+@pytest.mark.parametrize(
+    "atom", ["atom_018_Ar_N18_M1_uhf_def2svpd_g09.fchk", "atom_001_H_N02_M1_uhf_def2svpd_g09.fchk"]
+)
+def test_eval_radial_d_density(atom):
+    # conditional import of dev modules needed for this test
+    try:
+        from gbasis.wrappers import from_iodata
+        from gbasis.evals.density import evaluate_density_gradient
+        from grid import UniformInteger, LinearInfiniteRTransform, AtomGrid
+        from iodata import load_one
+        from atomdb.datasets.gaussian import eval_radial_d_density
+
+        assert True
+    except ImportError:
+        pytest.skip("Gbasis or IOData modules not available, skipping test")
+
+    # create atomic grid from 0 to 20 bohr
+    oned = UniformInteger(npoints=100)
+    rgrid = LinearInfiniteRTransform(1e-4, 20).transform_1d_grid(oned)
+    atgrid = AtomGrid(rgrid, degrees=[10])
+
+    # load the fchk file
+    mol_data = load_one(TEST_DATAPATH + "gaussian/" + atom)
+    ao_basis = from_iodata(mol_data)
+
+    # one electron RDM from fchk file
+    rdm = mol_data.one_rdms["scf"]
+
+    # evaluate gradient of the electron density on the grid
+    rho_grad = evaluate_density_gradient(rdm, ao_basis, atgrid.points)
```

Comment:
Why not store the variables `rdm`, `ao_basis` and `points` and the expected answer rho_grad in a [Numpy .npz](https://numpy.org/doc/stable/reference/generated/numpy.savez.html) file for each test case, and store that instead of the .fchk? It may remove some of the dependencies on the external  packages, although not the one on GBasis. Or is it not worth the trouble in this case?

Reply:
Ok, I think this is a good idea. I am on it.